### DISCUSSION TITLE
Where is the URDF?

### DISCUSSION THREAD

**Discussion by ChenyangRan** [robot-descriptions/robot_descriptions.py#90] [Q&A] [answered]
Hi, your work is really helpful. But I need the urdf path transfer parameter, could you please tell me where it is saved? I checked sit-packgs and did not find it.

**Comment by stephane-caron** [comment, marked as answer]
The way to get the URDF path currently is to import the robot description as a submodule and get the path from it, for example:

```py
$ python
Python 3.12.3 | packaged by conda-forge | (main, Apr 15 2024, 18:38:13) [GCC 12.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from robot_descriptions import jvrc_description
>>> jvrc_description.URDF_PATH
'/home/your_user/.cache/robot_descriptions/jvrc_description/urdf/jvrc1.urdf'
```

Unlike [`example-robot-data`](https://pypi.org/project/example-robot-data/) where all descriptions are packaged and installed to site-packages, in `robot_descriptions` the descriptions are cloned to a local cache at runtime. You can set the location of this cache via the ``ROBOT_DESCRIPTIONS_CACHE`` environment variable.
